src/index.html

```diff
@@ -7,7 +7,54 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </div>
+
+    <div class="stars stars--1">
+      <span class="stars__star active"></span>
```

Comment:
```suggestion
      <span class="stars__star"></span>
```
![image](https://github.com/mate-academy/layout_stars/assets/96409563/0f2157c3-e26a-402e-a1a0-55111a33b421)